WolvenKit.Modkit/RED4/Tools/MorphTargetTools.cs

```diff
@@ -76,9 +85,6 @@ public bool ExportMorphTargets(
                 }
                 expTargets.Add(ContainRawTarget(diffsBuffer, mappingBuffer, tempNumVertexDiffsInEachChunk, tempNumVertexDiffsMappingInEachChunk, targetsInfo.TargetStartsInVertexDiffs[i], targetsInfo.TargetStartsInVertexDiffsMapping[i], targetsInfo.TargetPositionDiffOffset[i], targetsInfo.TargetPositionDiffScale[i], expMeshes.Count));
             }
-
-            // Only add textures to list if the setting is checked in the export
-            var textureStreams = exportTextures ? ContainTextureStreams(blob, texBuffer) : [];
```

Comment:
Aren't these textures used for anything or why we can ignore them?

Reply:
They were only used for import in the Blender addon for morphtargets. I guess if we yeet them, they can rip out that part of their code as well